scss/_variables.scss

```diff
@@ -1272,7 +1272,6 @@ $tooltip-border-radius:             $border-radius !default;
 $tooltip-opacity:                   .9 !default;
 $tooltip-padding-y:                 $spacer * .25 !default;
 $tooltip-padding-x:                 $spacer * .5 !default;
-$tooltip-margin:                    0 !default;
```

Comment:
This would need to get nullified and deprecated, not removed outright.

Reply:
@mdo I have push a commit, with the nullified variable and the deprecation message. 

TBH I am not sure, I 've done it the right way, so feel free to change what ever you like